ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/PartyAllocationResult.scala

```diff
@@ -16,19 +16,30 @@ object PartyAllocationResult {
     override def description: String = "Party successfully allocated"
   }
 
+  /** The system is overloaded, clients should back off exponentially */
+  final case object Overloaded extends PartyAllocationResult {
+    override def description: String = "System is overloaded, please try again later"
```

Comment:
```suggestion
    override val description: String = "System is overloaded, please try again later"
```
Not sure it makes sense to wrap a constant around a method.